tests/test_blocks.py

```diff
@@ -319,7 +319,8 @@ def test_Quiver(self):
 
     @animation_compare(baseline_images="Blocks/Surface", nframes=3)
     def test_Surface(self):
-        if mpl._version.version_tuple < (3, 9):
+        mpl_version = tuple([int(x) for x in mpl._version.version.split(".")[:3]])
```

Comment:
You may want to use `packaging.version.parse` as is used in other tests instead.

Reply:
For example, this is now breaking with 3.10.0rc1:
```
    @animation_compare(baseline_images="Blocks/Surface", nframes=3)
    def test_Surface(self):
>       mpl_version = tuple([int(x) for x in mpl._version.version.split(".")[:3]])
E       ValueError: invalid literal for int() with base 10: '0rc1'
```

Reply:
Sorry for not fixing earlier:
https://github.com/boutproject/animatplot-ng/commit/844ab4443c8987923b2a2f4d743bbd237d917028
Do you want me to push the fix to fedora? Is rawhide sufficient?

Reply:
This test is the only failing one, so I suspect it will be fine:
https://copr.fedorainfracloud.org/coprs/qulogic/matplotlib-310/build/8245763/
But a build in Rawhide would be handy to check, or I could build the patch in this COPR.